src/traces/sankey/render.js

```diff
@@ -688,6 +813,9 @@ module.exports = function(gd, svg, calcData, layout, callbacks) {
         firstRender = true;
     });
 
+    // To prevent animation on dragging
+    var dragcover = gd.querySelector('.dragcover');
```

Comment:
Can we use `gd._fullLayout._dragCover` here? That `querySelector()` could be slow.

Reply:
Yes we can! Done in 40fb223fe48cacb155f294193fedbffc15a93069